### PR TITLE
feature-khweb-1/add-about-page

### PR DESCRIPTION
**Pull request by topsart**
[Add About page](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=3370&projectKey=KHWEB&view=detail&selectedIssue=KHWEB-1)

- added structure, styles, team section, desription section, team info

### CODE REVIEW

File: src/components/AboutPage/components/AboutDesription/AboutDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './AboutDescription.scss';
+
+export const AboutDesription = () => {
+  return (
+    <>
```
Comment (Aleksandra05sh) [inline src/components/AboutPage/components/AboutDesription/AboutDescription.tsx]: I think better to change <> on <section>

File: src/components/AboutPage/components/AboutList/components/AboutItem/components/AboutItemActions/AboutItemActions.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { CardActions, Button } from '@mui/material';
+import type { AboutContacts } from '../../../../../../types/About';
+import './AboutItemActions.scss';
+
+interface AboutItemActionsProps {
+  contacts: AboutContacts[];
+}
+
+export const AboutItemActions = ({ contacts }: AboutItemActionsProps) => (
```
Comment (Aleksandra05sh) [inline src/components/AboutPage/components/AboutList/components/AboutItem/components/AboutItemActions/AboutItemActions.tsx:10]: I think we decided to write in this way: 
`export const AboutItemActions React.FC<AboutItemActionsProps>  = ({ contacts }: AboutItemActionsProps) .....`

File: src/components/AboutPage/components/AboutList/components/AboutItem/components/AboutItemContent/AboutItemContent.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  name: string;
+  position: string;
+  description: string;
+}
+
+export const AboutItemContent = ({
+  name,
+  position,
+  description,
+}: AboutItemContentProps) => (
```
Comment (Aleksandra05sh) [inline src/components/AboutPage/components/AboutList/components/AboutItem/components/AboutItemContent/AboutItemContent.tsx:15]: the same
React.FC<AboutItemContentProps>

File: src/components/AboutPage/components/AboutDesription/AboutDescription.tsx (hunk 1 above)
Comment (topsart) [reply src/components/AboutPage/components/AboutDesription/AboutDescription.tsx]: Yes, i've removed it in the next commit

File: src/components/AboutPage/components/AboutDesription/AboutDescription.scss [hunk 4]
```diff
@@ -1,14 +1,21 @@
 @import '../../../../shared/styles/variables';
 @import '../../../../shared/styles/mixins';
 
+.about-description {
+  padding: 1rem 2rem 1rem;
```
Comment (Aleksandra05sh) [inline src/components/AboutPage/components/AboutDesription/AboutDescription.scss:5]: you can write in this case: `padding: 1rem 2rem`

File: src/components/AboutPage/components/AboutDesription/AboutDescription.tsx (hunk 1 above)
Comment (Aleksandra05sh) [reply src/components/AboutPage/components/AboutDesription/AboutDescription.tsx]: `<div>` is Ok, but for semantic markup, I think `<section>` will be better

File: src/components/AboutPage/components/AboutList/components/AboutItem/components/AboutItemActions/AboutItemActions.tsx (hunk 2 above)
Comment (topsart) [reply src/components/AboutPage/components/AboutList/components/AboutItem/components/AboutItemActions/AboutItemActions.tsx:10]: I didn't use this in the Astronauts page.
Maybe i missed something...

File: src/components/AboutPage/components/AboutDesription/AboutDescription.scss (hunk 4 above)
Comment (topsart) [reply src/components/AboutPage/components/AboutDesription/AboutDescription.scss:5]: Thank's, fixed

File: src/components/AboutPage/components/AboutDesription/AboutDescription.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
+@import '../../../../shared/styles/variables';
+@import '../../../../shared/styles/mixins';
+
+.about-description {
+  padding: 1rem 2rem 1rem;
+  margin-left: 1.65rem;
+  margin-right: 1.65rem;
+  border-radius: 4px;
```
Comment (vovkis13) [inline src/components/AboutPage/components/AboutDesription/AboutDescription.scss:8]: There is a variable `$default-border-radius` with such value

File: src/components/AboutPage/components/AboutList/AboutList.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Grid } from '@mui/material';
+import { AboutItem } from './components/AboutItem';
+import { aboutData } from '../../mocks/aboutData';
+import { About } from '../../types/About';
```
Comment (vovkis13) [inline src/components/AboutPage/components/AboutList/AboutList.tsx:5]: add `type`

File: src/components/AboutPage/components/AboutList/components/AboutItem/AboutItem.tsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Card } from '@mui/material';
+import { AboutItemImage } from './components/AboutItemImage';
+import { AboutItemContent } from './components/AboutItemContent';
+import { AboutItemActions } from './components/AboutItemActions';
+import { About } from '../../../../types/About';
```
Comment (vovkis13) [inline src/components/AboutPage/components/AboutList/components/AboutItem/AboutItem.tsx:6]: add `type`

File: src/components/AboutPage/components/AboutList/AboutList.tsx (hunk 6 above)
Comment (topsart) [reply src/components/AboutPage/components/AboutList/AboutList.tsx:5]: done

File: src/components/AboutPage/components/AboutList/components/AboutItem/AboutItem.tsx (hunk 7 above)
Comment (topsart) [reply src/components/AboutPage/components/AboutList/components/AboutItem/AboutItem.tsx:6]: done

File: src/components/AboutPage/components/AboutList/components/AboutItem/AboutItem.tsx [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+import './AboutItem.scss';
+
+export const AboutItem = ({
+  name,
+  position,
+  contacts,
+  description,
+  profile_image,
+}: About) => (
```
[the remark this answers is not in the document]
Comment (topsart) [reply src/components/AboutPage/components/AboutList/components/AboutItem/AboutItem.tsx:15]: fixed in all components with props

File: src/components/AboutPage/components/AboutDesription/AboutDescription.tsx [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+        quas, accusamus consectetur! Quam quis, at obcaecati, ex amet ipsum
+        numquam odio in vitae voluptatem fuga, error velit quaerat delectus
+        necessitatibus excepturi libero tenetur praesentium laboriosam voluptas
+        ad? Sapiente, perferendis suscipit expedita deserunt deleniti iusto
+        ipsum quas sit. Quos odio commodi repellat non, recusandae ipsum iusto,
+        necessitatibus delectus repellendus fugit voluptate enim cumque,
+        repudiandae harum vero veritatis blanditiis sit quia dicta? Unde, itaque
+        suscipit inventore enim id nobis nostrum maiores, autem dolorem ducimus
+        eligendi.
+      </p>
```
Comment (kerrimov) [inline src/components/AboutPage/components/AboutDesription/AboutDescription.tsx:21]: Use a more appropriate description

File: src/components/AboutPage/components/AboutDesription/AboutDescription.tsx [hunk 10]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './AboutDescription.scss';
+
+export const AboutDesription = () => {
+  return (
```
Comment (kerrimov) [inline src/components/AboutPage/components/AboutDesription/AboutDescription.tsx:5]: no need for return

File: src/components/AboutPage/components/AboutList/AboutList.scss [hunk 11]
```diff
@@ -0,0 +1,13 @@
+.about-list {
+  margin-bottom: 3rem;
+}
+
+.about-list-grid {
+  &:nth-child(1) {
+    margin-left: auto;
+  }
+  &:nth-child(2) {
+    margin-right: 23.26rem;
```
Comment (kerrimov) [inline src/components/AboutPage/components/AboutList/AboutList.scss:10]: 23.26?

File: src/components/AboutPage/mocks/aboutData.ts [hunk 12]
```diff
@@ -0,0 +1,95 @@
+import { GitHub, Instagram, LinkedIn } from '@mui/icons-material';
+import type { About } from '../types/About';
+
+export const aboutData: Array<About> = [
+  {
+    name: 'Togrul Kerrimov',
+    position: 'Product Owner',
```
Comment (kerrimov) [inline src/components/AboutPage/mocks/aboutData.ts:7]: I am team lead or technical expert

File: src/components/AboutPage/mocks/aboutData.ts [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+  {
+    name: 'Vladyslav Khrychev',
+    position: 'Team mentor',
+    contacts: [
+      {
+        icon: Instagram,
+        url: 'https://instagram.com/aleksandra05sh?igshid=YmMyMTA2M2Y=',
+      },
+      {
+        icon: LinkedIn,
```
Comment (kerrimov) [inline src/components/AboutPage/mocks/aboutData.ts:31]: Why Aleksandra's social is used for me and Vlad?

File: src/components/AboutPage/components/AboutDesription/AboutDescription.tsx (hunk 9 above)
Comment (kerrimov) [reply src/components/AboutPage/components/AboutDesription/AboutDescription.tsx:21]: It seems to me, that Typography should be used here

File: src/components/AboutPage/components/AboutDesription/AboutDescription.tsx (hunk 10 above)
Comment (topsart) [reply src/components/AboutPage/components/AboutDesription/AboutDescription.tsx:5]: fixed

File: src/components/AboutPage/components/AboutList/AboutList.scss (hunk 11 above)
Comment (topsart) [reply src/components/AboutPage/components/AboutList/AboutList.scss:10]: This is a feature for creating two rows with different number of elements in it.
In this case 1st row - 2 elements
2nd - 4
It's temporary solution will be fix in the future.

File: src/components/AboutPage/mocks/aboutData.ts (hunk 12 above)
Comment (topsart) [reply src/components/AboutPage/mocks/aboutData.ts:7]: fixed to Team Lead

File: src/components/AboutPage/mocks/aboutData.ts (hunk 13 above)
Comment (topsart) [reply src/components/AboutPage/mocks/aboutData.ts:31]: It was template.
Already fixed with real data.

File: src/components/AboutPage/components/AboutDesription/AboutDescription.tsx (hunk 9 above)
Comment (topsart) [reply src/components/AboutPage/components/AboutDesription/AboutDescription.tsx:21]: Typography added
Description is changed

**Review by kerrimov — DISMISSED** [review]

**Review by kerrimov — APPROVED** [review]

**Review by kerrimov — APPROVED** [review]

**Pull request merged**